portal-api/src/v1/validation/loan-rules/facility-stage-unconditional/requested-cover-start-date.js

```diff
@@ -17,56 +14,54 @@ module.exports = (submittedValues, deal, errorList) => {
     'requestedCoverStartDate-year': requestedCoverStartDateYear,
   } = submittedValues;
 
-  const requestedCoverStartDateTimestamp = formattedTimestamp(submittedValues.requestedCoverStartDate);
+  const requestedCoverStartDate = new Date(Number.parseInt(submittedValues.requestedCoverStartDate, 10));
 
   // EC 15 is: 'Cover Start Date is no more than three months from the date of submission'
   const eligibilityCriteria15 = deal.eligibility.criteria.find((c) => c.id === 15);
   const canEnterDateGreaterThan3Months = eligibilityCriteria15 && eligibilityCriteria15.answer === false;
 
   const dealHasBeenSubmitted = deal.details.submissionDate;
 
-  const nowDate = moment().startOf('day');
+  const nowDate = new Date();
 
   if (!dealHasBeenSubmitted) {
-    const {
-      coverDayValidation,
-      coverMonthValidation,
-      coverYearValidation
-    } = coverDatesValidation(requestedCoverStartDateDay, requestedCoverStartDateMonth, requestedCoverStartDateYear);
-
-    if (requestedCoverStartDateTimestamp) {
-      if (moment(requestedCoverStartDateTimestamp).isBefore(nowDate)) {
+    const { coverDayValidation, coverMonthValidation, coverYearValidation } = coverDatesValidation(
+      requestedCoverStartDateDay,
+      requestedCoverStartDateMonth,
+      requestedCoverStartDateYear,
+    );
+
+    if (submittedValues.requestedCoverStartDate) {
+      if (isBefore(requestedCoverStartDate, startOfDay(nowDate))) {
```

Comment:
Also same comments on this file as `supply-contract-conversion-rate-date-is-required-for-non-GBP.js`